### ISSUE TITLE
Craft 3.1 compatibility

### ISSUE THREAD

**Issue by boboldehampsink** [barrelstrength/craft-sprout-forms#220]
### Description

Craft 3.1 saves plugin settings in the Project Config while Sprout Forms tries to use the settings column on the plugin table. This column doesn't exist anymore in Craft 3.1.

See https://github.com/craftcms/cms/commit/254d1c14edcec0f2f7b2403d53f286fef7187244#diff-eb4ab0584b5426ababdb84f2ffadf519 and https://github.com/craftcms/cms/commit/a55f969e1c5c2ce6f4f16e209dc334f32f59958e#diff-eb4ab0584b5426ababdb84f2ffadf519

### Steps to reproduce

1. Install new Craft 3.1
2. Install new Sprout Forms

### Additional info

- Craft version: 3.1.3
- PHP version: 7.3.1
- Database driver & version: PostgreSQL 10.6
- Plugins & versions: Sprout Forms 3.0 beta

<img src="https://frontapp.com/assets/img/icons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_gskb)

**Comment by helleholmsen** [comment]
How long until the plugin works with the newest version of Craft?

**Comment by BenParizek** [comment]
@helleholmsen We're hoping to have an initial release with 3.1 support out later today.

**Comment by BenParizek** [comment]
Initial Craft 3.1 support has been added in Sprout Forms v3.0.0-beta.37

**Comment by wei2go** [comment]
Hi @BenParizek , I'm on Craft 3.1.10 and Sprout Forms v3.0.0-beta.41, but still getting a "Couldn't install plugin" error after hitting the 'Install' button on the plugin page. The plugin was installed successfully from plugin store. Any idea?

**Comment by BenParizek** [comment]
@wei2go Do you see any errors in the Craft logs (storage/logs/web.log)?

**Comment by wei2go** [comment]
> @wei2go Do you see any errors in the Craft logs (storage/logs/web.log)?

Yup, I can see:

`> create table {{%sproutforms_forms}} ...Exception: SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'sproutforms_forms' already exists`

Not sure why this happened. I could not update SproutForms (errors shown during update process, forgot what that was) after updating to Craft 3.1, so I removed the previous version and attempted another install from plugin store. So maybe some tables did not get removed for some reason.

Removed all sprout tables and did another install from plugin store, now it seems to work correctly.
